docs/docsite/rst/community/communication.rst

```diff
@@ -94,6 +94,36 @@ IRC meetings
 The Ansible community holds regular IRC meetings on various topics, and anyone who is interested is invited to
 participate. For more information about Ansible meetings, consult the `meeting schedule and agenda page <https://github.com/ansible/community/blob/master/meetings/README.md>`_.
 
+The Bullhorn
+------------
+The Bullhorn is our newsletter for the Ansible developer community. If you have any questions or content you’d like to share, please reach out to us at the-bullhorn@redhat.com, or directly contribute/suggest content for upcoming issues. The Ansible News Working Group sources, curates, and promotes existing Ansible-related content, and encourages the creation of Ansible-related content across a variety of mediums. Initial curation and promotion goals are an Ansible Weekly Newsletter; future work may include Ansible Magazine (online site, Ansible-community created / edited); Ansible Developer Newsletter (less frequent, info that is relevant to contributors).
+
+Subscribe to The Bullhorn here.
+
+`Issue #11 <https://bit.ly/thebullhorn11>`_
```

Comment:
This list will quickly get out of date could we delete the list of issues?

Maybe we could just point to the archive https://us19.campaign-archive.com/home/?u=56d874e027110e35dea0e03c1&id=d6635f5420